.github/workflows/integration.yml

```diff
@@ -17,6 +17,7 @@ on:
 
 jobs:
   pre_check:
+    if: ${{ github.repository_owner == 'agoric' }}
```

Comment:
I'm surprised this job requires credentials

Reply:
Ah it's for the notifications!
Let's move the check on the notify on failure step only

Reply:
Sure; done.